make-util.js

```diff
@@ -170,7 +182,15 @@ var buildNodeTask = function (taskPath, outDir) {
         cd(taskPath);
     }
 
-    run('tsc --outDir "' + outDir + '" --rootDir "' + taskPath + '"');
+    if (tscPath) {
```

Comment:
Can we add a comment here saying something along the lines of: use tsc if provided by task otherwise use globally set tsc.